cumulus_library/cli.py

```diff
@@ -47,18 +47,30 @@ class StudyBuilder:
     def __init__(  # pylint: disable=too-many-arguments
         self, region: str, workgroup: str, profile: str, schema: str
     ):
+        connect_kwargs = {}
+        for aws_env_name in [
+            "AWS_ACCESS_KEY_ID",
+            "AWS_SECRET_ACCESS_KEY",
+            "AWS_SESSION_TOKEN",
+        ]:
+            if aws_env_val := os.environ.get(aws_env_name):
+                connect_kwargs[aws_env_name.lower()] = aws_env_val
```

Comment:
pyathena doesn't grab those variables itself as fallbacks? I would have guessed that it would, but maybe it considers itself lower-level than that.

Reply:
yeah it doesn't, i've just verified. 🤷